frontend/src/reducers/plannerReducer.js

```diff
@@ -361,3 +354,46 @@ const plannerReducer = (state = initialState, action) => {
 };
 
 export default plannerReducer;
+
+// dummyMap.set("COMP2521", {
+//   title: "Data Structures and Algorithms",
+//   type: "Core",
+//   termsOffered: ["T0", "T1", "T2", "T3"],
+//   prereqs: "COMP1511 && (COMP1521 || DEFAULT3000)",
+//   plannedFor: null,
```

Comment:
why not delete?

Reply:
I've now deleted all but one. I kept one just so its clear what the structure of the course is.

Reply:
Oh my bad I didn't notice that this was documenting how to use the the function - that's fine.